sw/device/lib/dif/autogen/dif_sram_ctrl_autogen.inc

```diff
@@ -0,0 +1,43 @@
+// Copyright lowRISC contributors.
```

Comment:
I suppose this file needs to be deleted? Seems identical to `dif_sram_ctrl_autogen.h`.

Reply:
yes that is correct, should be deleted. *.inc was deprecated in favor of *.h in #8213.